lua/framework/physics/lua_material.h

```diff
@@ -115,13 +115,21 @@ int MatAddProperty(lua_State* L);
  * ####_
  *
  * OPENSN_XSFILE\n
- * Loads transport cross-sections from OpenSn cross-section files. Expects
+ * Loads transport cross sections from OpenSn cross-section files. Expects
  * to be followed by a filepath specifying the xs-file.
  *
  * ####_
  *
+ * OPENMC_XSLIB\n
+ * Loads transport cross sections from OpenMC cross-section files. Expects
```

Comment:
see, here you call it OpenMC xs **file** in the comment. So LIB or FILE?

Reply:
so change the comment: files -> libraries if you prefer to keep XSLIB

Reply:
I'll change this to library.

Reply:
Fixed.